File: problem-113/solution.go
package problem113

import "strings"

// reverseWords reverses given string of space separated words and return the reversed order of the words.
func reverseWords(s string) string {
	words := strings.Split(s, " ")

	left, right := 0, len(words)-1
	for left < right {
		words[left], words[right] = words[right], words[left]
		left++
		right--
	}

	return strings.Join(words, " ")
}

File: problem-113/solution.py
from unittest import TestCase


def reverse_words(s: str) -> str:
    words = list(reversed(s.split(" ")))
    return " ".join(words)


class Test(TestCase):
    def test(self):
        s = "hello world here"
        expected = "here world hello"
        got = reverse_words(s)
        self.assertEqual(expected, got)
